source/AI.cpp

```diff
@@ -2868,16 +2858,12 @@ void AI::AimTurrets(const Ship &ship, FireCommand &command, bool opportunistic)
 	const Ship *currentTarget = ship.GetTargetShip().get();
 	if(opportunistic || !currentTarget || !currentTarget->IsTargetable())
 	{
-		// Find the maximum range of any of this ship's turrets.
-		double maxRange = 0.;
-		for(const Hardpoint &weapon : ship.Weapons())
-			if(weapon.CanAim())
-				maxRange = max(maxRange, weapon.GetOutfit()->Range());
+		// Find the maximum range of any of this ship's turrets, and extend
+		// the weapon range slightly to account for velocity differences.
```

Comment:
I wouldn't consider a factor 1.5 slight, but that's not made by you.

Reply:
Yes, probably could be better to have a different name, something like 'radiusOfInterest'...

I took also a note because probably it should consider other ships 'inbound speed' so that factor can be decreased or removed (though it's a matter for another PR...).

Reply:
@RisingLeaf , renamed to `rangeOfInterest` and changed a bit how it's calculated (so it's 'constified'). It's ok for you?